sidebars.js

```diff
@@ -1023,111 +1023,105 @@ module.exports = {
   "Self-Managed": [
     "self-managed/about-self-managed",
     {
-      Setup: [
+      type: "category",
+      label: "Get started",
+      link: {
+        type: "doc",
+        id: "self-managed/get-started/index",
+      },
+      items: [
+        "self-managed/setup/deploy/local/c8run",
+        "self-managed/setup/deploy/local/local-kubernetes-cluster",
+      ],
+    },
+    {
+      Install: [
         "self-managed/setup/overview",
-        "self-managed/setup/install",
-        "self-managed/setup/upgrade",
         {
-          type: "category",
-          label: "Deploy",
-          items: [
+          type: "doc",
+          label: "Helm",
+          id: "self-managed/setup/install",
+        },
+        "self-managed/setup/deploy/other/docker",
+        "self-managed/setup/deploy/local/manual",
+      ],
+    },
+    {
+      type: "category",
+      label: "Deploy",
+      link: {
+        type: "doc",
+        id: "self-managed/setup/index",
+      },
+      items: [
+        {
+          "Reference architecture": [
+            "self-managed/reference-architecture/reference-architecture",
+            "self-managed/reference-architecture/manual/manual",
```

Comment:
Is the manual option intentionally under reference arch? Can I get more context here?